File: Iniciante/3040/3040.py
def escolher(h, d, g):
    if (h>=200 and h<=300) and (d>=50) and (g>=150):
        return "Sim"
    return "Nao"

n = int(input())
for i in range(n):
    h,d,g = map(int, input().split())
    print(escolher(h,d,g))

File: Iniciante/3040/3040.cpp
#include <bits/stdc++.h>

using namespace std;

string escolher(int h, int d, int g){
    if ((h>=200 && h<=300) && (d>=50) && (g>=150)) return "Sim";
    return "Nao";
}

int main(){
    int n, h, d, g;
    cin >> n;
    for(int i = 0; i<n; i++){
        cin >> h >> d >> g;
        cout << escolher(h,d,g) << endl;
    }
    return 0;
}
